### PR TITLE
fix(docs): Bring back the accidentally deleted client-metrics.md content.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
A lot of content was accidentally deleted in #4370, this brings back the content.

### CODE REVIEW

File: app/scripts/models/user.js [hunk 1]
```diff
@@ -69,6 +69,14 @@ define(function (require, exports, module) {
       });
     },
 
+    /**
+     * Log the number of stored accounts
+     */
+    logNumStoredAccounts () {
```
Comment (vladikoff) [inline app/scripts/models/user.js]: bad rebase?

Comment (shane-tomlinson) [inline app/scripts/models/user.js]: Yes.

**Comment by shane-tomlinson** [comment]
@vladikoff - mind another r?

**Comment by vladikoff** [comment]
Yay!

**Pull request merged**